### ISSUE TITLE
Bug: NordVPN United Kingdom Region no longer accepted

### ISSUE THREAD

**Issue by AdamHebby** [qdm12/gluetun#1674]
### Is this urgent?

No

### Host OS

Debian Bullseye

### CPU arch

aarch64

### VPN service provider

NordVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2023-06-08T22:48:56.392Z (commit ea19914)

### What's the problem 🤔

Gluetun no longer accepts "United Kingdom" as a server region for NordVPN, despite there being servers configured for United Kingdom.

```
> docker exec -it gluetunvpn /bin/sh
> cat /gluetun/servers.json | grep -ic '"region": "United Kingdom"'
27
```

I've switched to Europe for the time being, but this did use to work and I would prefer to use a UK server. I'll probably hard-code the tag to v3 for now.

### Share your logs

```plain text
Running version latest built on 2023-06-08T22:48:56.392Z (commit ea19914)
...
2023-06-10T13:24:12+01:00 INFO [routing] default route found: interface eth0, gateway 192.168.90.1, assigned IP 192.168.90.163 and family v4
2023-06-10T13:24:12+01:00 INFO [routing] local ethernet link found: eth0
2023-06-10T13:24:12+01:00 INFO [routing] local ipnet found: 192.168.90.0/24
2023-06-10T13:24:12+01:00 INFO [firewall] enabling...
2023-06-10T13:24:12+01:00 INFO [firewall] enabled successfully
2023-06-10T13:24:14+01:00 INFO [storage] merging by most recent 17678 hardcoded servers and 17678 servers read from /gluetun/servers.json
2023-06-10T13:24:14+01:00 ERROR VPN settings: provider settings: server selection: for VPN service provider nordvpn: the region specified is not valid: value is not one of the possible choices: value united kingdom, choices available are Africa, the Middle East and India, Asia Pacific, Europe, The Americas
```

### Share your configuration

```yml
gluetun:
    image: qmcgaw/gluetun:latest
    container_name: gluetunvpn
    cap_add:
      - NET_ADMIN
    network_mode: bridge
    networks:
      t2_proxy:
        ipv4_address: 192.168.90.163
    ports:
      - 8881:8881/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks
      - 8000:8000/tcp # Built-in HTTP control server
      # [REDACTED]
    volumes:
      - ./appdata/gluetun:/gluetun
      - ./appdata/gluetun/port_forward:/tmp/gluetun/forwarded_port
    environment:
      VPN_SERVICE_PROVIDER: nordvpn
      TZ: $TZ
      OPENVPN_USER: $NORDVPN_USERNAME
      OPENVPN_PASSWORD: $NORDVPN_PASSWORD
      SERVER_REGIONS: "United Kingdom"
      FIREWALL_OUTBOUND_SUBNETS: "192.168.0.0/24"
      HTTPPROXY: "enabled"
      HTTPPROXY_LISTENING_ADDRESS: ":8881"
      SHADOWSOCKS: "enabled"
      SHADOWSOCKS_PASSWORD: "[REDACTED]"
    restart: unless-stopped
```

**Comment by AdamHebby** [comment]
Switched to v3 and still doesn't accept UK is a valid region. This was working up until 2 days ago so not sure why v3 doesn't work, considering it hasn't been updated in 10 days.

I wonder if the NordVPN config has changed?

**Comment by jeffsauer** [comment]
Similar issue for me... can no longer use "United States" and had to change to "The Americas" as recommended by the error message.  However, I would prefer a US server.

**Comment by qdm12** [comment]
Duplicate of #1665

See https://github.com/qdm12/gluetun/issues/1665#issuecomment-1583557626

v3 images should be unaffected by this, please double check. You might be running v3.34.2 which was a bad release (see release notes), try repulling v3 image.

Let's keep this opened, I'll try making this retro-compatible before next release

**Comment by qdm12** [comment]
Fixed retro-compatibility in 4a0f9c36ba2aa5055f16f2444cc83c66521e7c48

**Comment by chenks** [comment]
what's changed and what do i need to change to get it to work as it was previously?
i was set to use Netherlands, and that's what it always connected to, now it's connecting to a random country.

this is what i have always had set in the config
SERVER_REGIONS=Netherlands

do i now need to change this to
SERVER_REGIONS=Europe
SERVER_COUNTRIES=Netherlands

i tried to find reference to this in the WIKI but couldn't see anything about it, maybe didn't look in the correct place.
i also aren't seeing anything in the logs about specified region not being valid.
